staking/apr/compute.go

```diff
@@ -27,22 +29,21 @@ type Reader interface {
 }
 
 const (
-	secondsInYear = int64(31_557_600)
+	secondsInYear = int64(31557600)
 )
 
 var (
 	oneYear = big.NewInt(int64(secondsInYear))
 )
 
 func expectedRewardPerYear(
-	oneEpochAgo, twoEpochAgo *block.Header,
-	oneSnapshotAgo, twoSnapshotAgo *staking.ValidatorWrapper,
-	blocksPerEpoch uint64,
+	now, oneEpochAgo *block.Header,
+	curValidator, snapshotLastEpoch *staking.ValidatorWrapper,
 ) (*big.Int, error) {
-	oneTAgo, twoTAgo := oneEpochAgo.Time(), twoEpochAgo.Time()
+	timeNow, oneTAgo := now.Time(), oneEpochAgo.Time()
 	diffTime, diffReward :=
-		new(big.Int).Sub(twoTAgo, oneTAgo),
-		new(big.Int).Sub(twoSnapshotAgo.BlockReward, oneSnapshotAgo.BlockReward)
+		new(big.Int).Sub(timeNow, oneTAgo),
+		new(big.Int).Sub(curValidator.BlockReward, snapshotLastEpoch.BlockReward)
```

Comment:
do we need to take the absolute value here? as `diffReward` can be negative?

Reply:
diffReward can not. because reward is increasing only.

Reply:
true. I got confused with block reward.